.github/workflows/container.yml

```diff
@@ -148,9 +148,9 @@ jobs:
         shell: bash
         run: |
           ls -lR bpf
-          mkdir -p pkg/profiler/cpu/bpf
-          cp -r bpf/out/* pkg/profiler/cpu/bpf
-          rm pkg/profiler/cpu/bpf/*/pid_namespace.bpf.o
+          mkdir -p pkg/profiler/cpu/bpf/programs
```

Comment:
If we are doing this now, let's wipe the `cpu` part of the path as it's wrong. These are unwinders, the profiler type is directed by where are we attaching these programs

Reply:
So you want me to rename `pkg/profiler/cpu` to `pkg/profiler/unwinder` ?

Reply:
That name works. This is somethign I wanted to do for the bpf progrs too in `bpf/` but let's try to focus this week on reliability 😄

Reply:
Let's talk about this later. We can change it to whatever we think is meaningful.